cmd/lifecycle/exporter.go

```diff
@@ -205,31 +206,46 @@ func (e *exportCmd) export(group buildpack.Group, cacheStore lifecycle.Cache, an
 		return err
 	}
 
-	report, err := exporter.Export(lifecycle.ExportOptions{
-		AdditionalNames:    e.AdditionalTags,
-		AppDir:             e.AppDir,
-		DefaultProcessType: e.DefaultProcessType,
-		ExtendedDir:        e.ExtendedDir,
-		LauncherConfig:     launcherConfig(e.LauncherPath, e.LauncherSBOMDir),
-		LayersDir:          e.LayersDir,
-		OrigMetadata:       analyzedMD.LayersMetadata,
-		Project:            projectMD,
-		RunImageRef:        runImageID,
-		RunImageForExport:  runImageForExport,
-		WorkingImage:       appImage,
-	})
+	var wg sync.WaitGroup
+	var report files.Report
+	err = nil
```

Comment:
nit: I think this might be redundant... you could also call it `exportErr` or something, to distinguish it from the other var

Reply:
Great idea! I renamed it to `appErr` because I saw there is already a `cacheErr`.